ubuntu/14.04/standard/Dockerfile

```diff
@@ -62,12 +71,16 @@ RUN apt-get update \
 RUN apt-get update \
  && apt-get install -y --no-install-recommends openjdk-8-jdk \
  && rm -rf /var/lib/apt/lists/*
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends openjdk-9-jdk \
```

Comment:
Last time we checked, there was not a openjdk-9-jdk package available on Ubuntu 14.04. This is currently being installed on the Ubuntu 16.04 image only via the ADDITIONAL_JDK_VERSION replacement. Can you confirm that the 14.04 image builds, and if so, we should remove the ADDITIONAL_JDK_VERSION replacement token since it was essentially a hack to get JDK 9 installed on 16.04 only.

Reply:
Good catch. I had tested the script on one of the Ubuntu 16.04 images and missed that it wasn't available for the 14.04 images. I'll revert the change.